src/StaticContentHandler.cpp

```diff
@@ -98,22 +104,22 @@ HTTPResponse StaticContentHandler::handleNotFound(void)
 
 	// Create a simple HTML body for the 404 page
 	std::string htmlBody = "<!DOCTYPE html>\n"
-						   "<html lang=\"en\">\n"
-						   "<head>\n"
-						   "    <meta charset=\"UTF-8\">\n"
-						   "    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\n"
-						   "    <title>404 Not Found</title>\n"
-						   "    <style>\n"
-						   "        body { font-family: Arial, sans-serif; text-align: center; margin-top: 50px; }\n"
-						   "        h1 { color: #ff6347; }\n"
-						   "        p { color: #757575; }\n"
-						   "    </style>\n"
-						   "</head>\n"
-						   "<body>\n"
-						   "    <h1>404 Not Found</h1>\n"
-						   "    <p>Sorry, the page you're looking for cannot be found.</p>\n"
-						   "</body>\n"
-						   "</html>";
+						"<html lang=\"en\">\n"
```

Comment:
Do we have alreday an issue to server the home page and the errors the propert way? It should not be done in this PR, but since I'm reading it we could already write an issue, no?

Reply:
regarding the errors we have this one:
https://linear.app/web-serv/issue/WEB-127/return-response-according-to-parser-status